firmware/library/L3_HAL/apds_9960.hpp

```diff
@@ -362,14 +353,12 @@ class Apds9960 : public Apds9960Interface
         // then that photodiode was covered and movement detected
         if ((gfifo_data_[i] - gfifo_data_[i + 1]) < up_sensitivity_)  // Up
         {
-          points_[index_] = Direction::kUp;
-          index_++;
+          points_.push_back(Direction::kUp);
```

Comment:
There isn't a check for ponts_.size() anywhere here, so what prevents it from overflowing?